src/components/PricePlan.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import { PlanType, PriceType } from '../types';
+import icon from '../assets/icon-music.svg';
+
+interface Props {
+  plan: PlanType;
+  selected: boolean;
+  onHandleClick: (val: string) => void;
+}
+
+type ButtonProps = {
```

Comment:
Not sure what purpose this type serves here..

Reply:
It was to do with the StyledComponent, but it looks like it is redundant now so will remove